packages/jupyter-ai/jupyter_ai/extension.py

```diff
@@ -358,11 +362,11 @@ def _init_chat_handlers(self):
             "llm_chat_memory": self.settings["llm_chat_memory"],
             "root_dir": self.serverapp.root_dir,
             "dask_client_future": self.settings["dask_client_future"],
-            "model_parameters": self.settings["model_parameters"],
```

Comment:
This is unrelated cleanup, it was included twice here (look 5 lines above) and my IDE was screaming at this dict because of it

Reply:
Thanks for clarifying.